### PR TITLE
https://issues.redhat.com/browse/ACM-13270

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: release_notes/whats_new.adoc [hunk 1]
```diff
@@ -79,6 +79,9 @@ See link:../console/console.adoc#home-page[Console overview].
 
 * To configure a cluster based on the available node roles, you can now use the `getNodesWithExactRoles` function to receive a list of nodes, and use the `hasNodesWithExactRoles` function to receive confirmation about clusters that contain nodes with only the roles that you specified. See link:../governance/template_functions.adoc#template-functions[Template functions] for more details.
 
+
+* You can now view the metadata of a root policy by using the `.PolicyMetadata` variable. See the link:../governance/template_support_intro.adoc#template-coomparison-table[Comparison of hub cluster and managed cluster templates].
```
Comment (mprahl) [inline release_notes/whats_new.adoc]:
```suggestion
* You can now view the metadata of a root policy by using the `.PolicyMetadata` hub template variable. See the link:../governance/template_support_intro.adoc#template-coomparison-table[Comparison of hub cluster and managed cluster templates].
```

**Review by mprahl — APPROVED** [review]

**Review by drkho — APPROVED** [review]

File: governance/template_support_intro.adoc [hunk 2]
```diff
@@ -57,6 +57,10 @@ The equivalent call might use the following syntax: `{{hub "(lookup "v1" "Secret
 | The output of template functions are stored in `Policy` resource objects in each applicable managed cluster namespace on the hub cluster, before it is synced to the managed cluster. This means that any sensitive results from template functions are readable by anyone with read access to the `Policy` resource objects on the hub cluster, and anyone with read access to the `ConfigurationPolicy` or `OperatorPolicy` resource objects on the managed clusters. Additionally, if etcd encryption is enabled, the policy resource objects are not encrypted. It is best to carefully consider this when using template functions that return sensitive output (e.g. from a secret).
 | The output of template functions are not stored in policy related resource objects.
 
+| Policy metadata
```
Comment (swopebe) [inline governance/template_support_intro.adoc:60]: I think the comma can be removed on line 61; question...do we need the `.` on the spec?

Typically I see that if using a key value pair in the same code, but for one only, I don't think I have ever seen it.

Approving with this comment.

**Review by swopebe — APPROVED** [review]

File: release_notes/whats_new.adoc [hunk 3]
```diff
@@ -79,6 +79,9 @@ See link:../console/console.adoc#home-page[Console overview].
 
 * To configure a cluster based on the available node roles, you can now use the `getNodesWithExactRoles` function to receive a list of nodes, and use the `hasNodesWithExactRoles` function to receive confirmation about clusters that contain nodes with only the roles that you specified. See link:../governance/template_functions.adoc#template-functions[Template functions] for more details.
 
+
+* You can now view the metadata of a root policy by using the `.PolicyMetadata` hub template variable. See the link:../governance/template_support_intro.adoc#template-coomparison-table[Comparison of hub cluster and managed cluster templates].
```
Comment (swopebe) [inline release_notes/whats_new.adoc]: hub _cluster_ template?

File: governance/template_support_intro.adoc [hunk 4]
```diff
@@ -57,6 +57,10 @@ The equivalent call might use the following syntax: `{{hub "(lookup "v1" "Secret
...
+| The `.PolicyMetadata` variable resolves to a map with the `name`, `namespace`, `labels`, and `annotations` keys, with values from the root policy.
```
Comment (dockerymick) [inline governance/template_support_intro.adoc]:
```suggestion
| The `.PolicyMetadata` variable resolves to a map with the `name`, `namespace`, `labels`, and `annotations` keys with values from the root policy.
```

File: governance/template_support_intro.adoc (hunk 2 above)
Comment (dockerymick) [reply governance/template_support_intro.adoc:60]: @mprahl is the period required? `.PolicyMetadata`

File: release_notes/whats_new.adoc (hunk 3 above)
Comment (dockerymick) [inline release_notes/whats_new.adoc]:
```suggestion
* You can now view the metadata of a root policy by using the `.PolicyMetadata` hub cluster template variable. See the link:../governance/template_support_intro.adoc#template-coomparison-table[Comparison of hub cluster and managed cluster templates].
```

File: governance/template_support_intro.adoc (hunk 2 above)
Comment (mprahl) [reply governance/template_support_intro.adoc:60]: Yes, it's required. Thanks for checking!

Comment (dockerymick) [reply governance/template_support_intro.adoc:60]: Thanks for confirming!

**Pull request merged**